src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime.Tests/Microsoft.Azure.CognitiveServices.LUIS.Tests.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\LUIS-Runtime\Microsoft.Azure.CognitiveServices.LUIS.csproj" />
+    <ProjectReference Include="..\Runtime\Microsoft.Azure.CognitiveServices.LUIS.csproj" />
```

Comment:
include the imports in LUIS.Tests project.
Remove lines 5,6
Remove lines 9-16 (all these references are implicitly added to each test project)
Add relevant properties about assemblyname, version etc to LUIS.Test project file (basically model this project file according to other test project files.